tcms/testruns/sqls.py

```diff
@@ -1,18 +1,18 @@
 CLONE_ENV_VALUES = '''
```

Comment:
This file is already covered by #139 and #135, better merge one of them otherwise you are duplicating work, which has already been done.